packages/frontend/src/view/pages/user/components/UserAssetTable.tsx

```diff
@@ -26,14 +26,21 @@ export interface UserAssetEntry {
 }
 
 export function UserAssetsTable(props: UserAssetsTableProps) {
+  const forcedActionLink = (entry: UserAssetEntry) =>
+    AssetId.check(entry.asset.hashOrId)
+      ? `/forced/new/perpetual/${
+          entry.vaultOrPositionId
+        }/${entry.asset.hashOrId.toString()}`
+      : `/forced/new/spot/${entry.vaultOrPositionId}`
+
   return (
     <Table
       fullBackground
       columns={[
         { header: <span className="pl-10">Name</span> },
         { header: 'Balance' },
         { header: props.type === 'PERPETUAL' ? 'Position' : 'Vault' },
-        { header: props.isMine ? 'Action' : '' },
+        ...(props.isMine ? [{ header: 'Action' }] : []),
```

Comment:
Why do you think that’s better?

Reply:
The previous approach would create column with empty header and cell, this approach will not create any column

Reply:
I'm aware of that. Is that the desired behaviour?